ui/packages/consul-ui/app/services/repository/token.js

```diff
@@ -53,21 +45,14 @@ export default class TokenService extends RepositoryService {
 
   @dataSource('/:partition/:ns/:dc/token/self/:secret')
   self(params) {
-    // TODO: Does this need ns passing through?
+    // This request does not need ns or partition passing through as its
+    // inferred from the token itself.
```

Comment:
I checked in with the team about this and confirmed that it does not need `ns` or `partition` passing, so I fixed this up while I was here and added some more information here to help future me when I next come here